bundle/manifests/app.kiegroup.org_kogitoruntimes.yaml

```diff
@@ -161,6 +161,9 @@ spec:
             probes:
               description: Configure liveness, readiness and startup probes for containers
               properties:
+                livenessPath:
```

Comment:
Wouldn't it be better to use existing `path` parameter (https://github.com/kiegroup/kogito-cloud-operator/blob/e90c2f2d0175afc38fe797893f7a9ff141ae7006/bundle/manifests/app.kiegroup.org_kogitoruntimes.yaml#L205-L207) instead of creating this new one? AFAIK they serve the same purpose.